community/modules/project/service-account/outputs.tf

```diff
@@ -14,47 +14,20 @@
  * limitations under the License.
  */
 
-output "email" {
```

Comment:
could you comment on why you are removing these vars?

Reply:
I am removing:

- all variables intended for the use case of creating N>1 service accounts
- anything that is duplicative of the service_account output object (e.g. the e-mail address at line 17)

Reply:
got it, thanks!